frontend/src/routes/login/Login.svelte

```diff
@@ -116,6 +116,6 @@
         </form>
     </div>
     <img src={logo} alt="Logo isn't working"><br>
-    <a href="">Help, I cannot log in</a><br>
+    <a href="../login">Help, I cannot log in</a><br>
```

Comment:
How this href work? What is it pointing to?

Reply:
It's currently a placeholder. When I left empty href there was some problems.

Reply:
If it is placeholder, you can replace it with `href="#"` or just delete, it isn't necessary ;)